diff --git a/.SRCINFO b/.SRCINFO
index e316f87..7db261b 100644
--- a/.SRCINFO
+++ b/.SRCINFO
@@ -1,6 +1,6 @@
 pkgbase = tor-browser
 	pkgdesc = Tor Browser Bundle: anonymous browsing using Firefox and Tor
-	pkgver = 12.0.6
+	pkgver = 12.0.7
 	pkgrel = 1
 	url = https://www.torproject.org/projects/torbrowser.html
 	install = tor-browser.install
@@ -26,8 +26,8 @@ pkgbase = tor-browser
 	optdepends = gst-libav: H.264 video
 	optdepends = libpulse: PulseAudio audio driver
 	optdepends = libnotify: Gnome dialog boxes
-	noextract = tor-browser-linux32-12.0.6_ALL.tar.xz
-	noextract = tor-browser-linux64-12.0.6_ALL.tar.xz
+	noextract = tor-browser-linux32-12.0.7_ALL.tar.xz
+	noextract = tor-browser-linux64-12.0.7_ALL.tar.xz
 	source = tor-browser.desktop.in
 	source = tor-browser.in
 	source = tor-browser.png
@@ -37,13 +37,13 @@ pkgbase = tor-browser
 	sha256sums = 1143d23e347605b498b3793992e84e95563efd94aa4da17837b37104a6d4a090
 	sha256sums = f25ccf68b47f5eb14c6fec0664c74f30ea9c6c58d42fc6abac3b64670aaa3152
 	sha256sums = 7b28b5dbe8ad573bb46e61b4d542b33e01ca240825ca640b4893fee6203b021f
-	source_i686 = https://dist.torproject.org/torbrowser/12.0.6/tor-browser-linux32-12.0.6_ALL.tar.xz
-	source_i686 = https://dist.torproject.org/torbrowser/12.0.6/tor-browser-linux32-12.0.6_ALL.tar.xz.asc
-	sha256sums_i686 = 9e3241e64eeb4314722fb7e853c7cb090c72e37749bd5dbaa0abd0f9fc3a534a
+	source_i686 = https://dist.torproject.org/torbrowser/12.0.7/tor-browser-linux32-12.0.7_ALL.tar.xz
+	source_i686 = https://dist.torproject.org/torbrowser/12.0.7/tor-browser-linux32-12.0.7_ALL.tar.xz.asc
+	sha256sums_i686 = 68b1d95050d96788bb057a0ce58c4faf39d8c3cd76fce1a6846f7bb7da243afb
 	sha256sums_i686 = SKIP
-	source_x86_64 = https://dist.torproject.org/torbrowser/12.0.6/tor-browser-linux64-12.0.6_ALL.tar.xz
-	source_x86_64 = https://dist.torproject.org/torbrowser/12.0.6/tor-browser-linux64-12.0.6_ALL.tar.xz.asc
-	sha256sums_x86_64 = 30bcbf4fc03eb9d6ac2135984b36aa5d21610373c9b06ec39ca246d1bf5462f0
+	source_x86_64 = https://dist.torproject.org/torbrowser/12.0.7/tor-browser-linux64-12.0.7_ALL.tar.xz
+	source_x86_64 = https://dist.torproject.org/torbrowser/12.0.7/tor-browser-linux64-12.0.7_ALL.tar.xz.asc
+	sha256sums_x86_64 = 968f88cba23b4b5355d44f4204fa898d1173b845c60bcd0d454525a597c6e705
 	sha256sums_x86_64 = SKIP
 
 pkgname = tor-browser
diff --git a/PKGBUILD b/PKGBUILD
index 27eb92e..12ec20e 100644
--- a/PKGBUILD
+++ b/PKGBUILD
@@ -17,7 +17,7 @@
 
 
 pkgname='tor-browser'
-pkgver='12.0.6'
+pkgver='12.0.7'
 pkgrel=1
 pkgdesc='Tor Browser Bundle: anonymous browsing using Firefox and Tor'
 url='https://www.torproject.org/projects/torbrowser.html'
@@ -70,9 +70,9 @@ sha256sums=('5dd2b61bd4edf4d1499a81127f97a1de7ec272a885df97331b61969a5a07f05f'
             '1143d23e347605b498b3793992e84e95563efd94aa4da17837b37104a6d4a090'
             'f25ccf68b47f5eb14c6fec0664c74f30ea9c6c58d42fc6abac3b64670aaa3152'
             '7b28b5dbe8ad573bb46e61b4d542b33e01ca240825ca640b4893fee6203b021f')
-sha256sums_i686=("$(_dist_checksum "${_tag_i686}")"
+sha256sums_i686=('68b1d95050d96788bb057a0ce58c4faf39d8c3cd76fce1a6846f7bb7da243afb'
                  'SKIP')
-sha256sums_x86_64=("$(_dist_checksum "${_tag_x86_64}")"
+sha256sums_x86_64=('968f88cba23b4b5355d44f4204fa898d1173b845c60bcd0d454525a597c6e705'
                    'SKIP')
 
 noextract=("${pkgname}-${_tag_i686}-${pkgver}_${_pkgsuffx}.tar.xz"
