diff --git a/PKGBUILD b/PKGBUILD
index 71c3695..78682f3 100644
--- a/PKGBUILD
+++ b/PKGBUILD
@@ -142,7 +142,7 @@ package() {
 
 	install -dm755 "${pkgdir}/usr/bin"
 	sed "${_sed_subst}" "${pkgname}.in" > "${pkgdir}/usr/bin/${pkgname}"
-	chmod +x "${pkgdir}/usr/bin/${pkgname}"
+	chmod +rx "${pkgdir}/usr/bin/${pkgname}"
 
 	install -Dm 644 "${pkgname}.png" "${pkgdir}/usr/share/pixmaps/${pkgname}.png"
 
